README.md

```diff
@@ -39,9 +37,137 @@ modules_that_allow_delete_on_deploy:
 - core
 ```
 
+## Commands / business logic
+
+We recommend using commands to encapsulate business rules. By following our recommendation, you will improve the consistency of your code, so it will be easy to onboard new developers to the project and easier to take over existing projects. The advantage of using this architecture is that it will be easy to re-use the command - you will be able to execute it both in a live web request, as well as a background job.
+
+We recommend placing your commands in `lib/commands` directory (the old way, before introducing `lib` directory, was `views/partials/lib/commands`)
```

Comment:
Sure we need to include 'the old way'?

Reply:
not sure - I've added this line before copying files from views/partials/lib, in the end decided it would be super cool to have all `lib` files in `app/lib`, so might not be necessary to add this line.

Reply:
```suggestion
We recommend placing your commands in `lib/commands` directory.
```